src/pyrokinetics/diagnostics/synthetic_highk_dbs.py

```diff
@@ -0,0 +1,670 @@
+import os
+import sys
+
+import matplotlib as mp
+import matplotlib.pyplot as plt
+import numpy as np
+import xarray as xr
+import xrft
+from numpy.typing import ArrayLike
+
+import pyrokinetics as pk
+from pyrokinetics import Pyro, template_dir
+from pyrokinetics.typing import PathLike
+
+
+class Synthetic_highk_dbs:
```

Comment:
More pythonic name would be something like:

```suggestion
class SyntheticHighkDBS:
```